test/manual/amp-base-carousel/basic.amp.html

```diff
@@ -17,7 +18,7 @@
 </head>
 <body>
   <h1>A Basic carousel, with looping</h1>
-  <amp-base-carousel id="carousel-1" width="300" height="200" layout="responsive" loop="true">
```

Comment:
Why did we remove the `id` here?

Reply:
The only reason it had an id was because I originally copied it from a carousel example that had an id as a target for an action. It isn't needed so I just removed it here.

Reply:
Oops, didn't see this was still needed below.